src/main.ts

```diff
@@ -292,6 +291,7 @@ export class Extension {
     readonly texMagician: TeXMagician
     readonly envPair: EnvPair
     readonly section: Section
+    readonly latexCommanderProvider: LaTeXCommanderProvider
```

Comment:
`class Extention` should not have providers. Remove `Extension.latexCommanderProvider`.